e2e/playwright/payUserFlow.spec.ts

```diff
@@ -0,0 +1,276 @@
+import { test, expect, type Page } from "@playwright/test";
+import { readFileSync } from "fs";
+import path from "path";
+import { SUCCESS_CARD_NUMBER } from "./constants";
+import { awaitForPaymentResponse } from "./payUtils/awaitForPaymentResponse";
+import { fillGovUkCardDetails } from "./payUtils/fillGovUkCardDetails";
+import {
+  createFlow,
+  createTeam,
+  deleteTestData,
+  getAnalyticsByFlowId,
+  gqlAdmin,
+  insertPublishedFlow,
+  insertTestUser,
+  prodGqlPublic,
+} from "./utils";
+
+test.setTimeout(90_000);
+
+const URL = "http://localhost:3000";
+
+const TEST_EMAIL = "test@test.com";
+const DEFAULT_WAIT_TIME = 300;
+const TEAM_SLUG = "buckinghamshire";
+const FLOW_SLUG = "apply-for-a-lawful-development-certificate";
+
+test.describe("Full user payment flow", async () => {
```

Comment:
this test description (& the associated file name here) are a little confusing for me: rather than "full user" do we actually mean "Full LDC application with payment" ?